charts/datadog/templates/_container-system-probe.yaml

```diff
@@ -103,9 +109,10 @@
     - name: runtime-compiler-output-dir
       mountPath: {{ .Values.datadog.systemProbe.runtimeCompilationAssetDir }}/build
       mountPropagation: {{ .Values.datadog.hostVolumeMountPropagation }}
+      readOnly: true
```

Comment:
According to the name of the volume, this seems to be the path where `system-probe` writes the result of the eBPF programs compilation.
That’s why I’d be surprised if `system-probe` doesn’t fail if it cannot write there.